google-cloud-storage/src/main/java/com/google/cloud/storage/StorageOptions.java

```diff
@@ -93,6 +94,18 @@ public Builder setStorageRetryStrategy(StorageRetryStrategy storageRetryStrategy
       return this;
     }
 
+    /**
+     * Override default enablement of invocation id added to x-goog-api-client header.
+     *
+     * @param storageInvocationIDEnabled a non-null Boolean to change enablement of invocation id
+     * @return the builder
+     */
+    public Builder setStorageInvocationIDEnabled(Boolean storageInvocationIDEnabled) {
+      this.storageInvocationIDEnabled =
+          requireNonNull(storageInvocationIDEnabled, "storageInvocationIDEnabled must be non null");
```

Comment:
Once we change from `Boolean` -> `boolean` we won't need this nonNull check anymore.